ui/apps/platform/src/Components/CheckboxTable.jsx

```diff
@@ -111,7 +111,15 @@ class CheckboxTable extends Component {
     render() {
         const { manual, ...rest } = this.props;
         const columns = this.addCheckboxColumns();
-        return <Table {...rest} columns={columns} manual={manual} setTableRef={this.setTableRef} />;
+        return (
+            <Table
+                {...rest}
+                columns={columns}
+                manual={manual}
+                setTableRef={this.setTableRef}
+                className="pf-v5-u-background-color-100"
```

Comment:
> margins around the table

Because only the paranoid survive, can you make temporary code change to artificially duplicate enough clusters to table extends below the fold and make sure that the layout is as indented?

Reply:
The description there was a little misleading. It would have been more accurate to say added a `PageSection` with padding around the table. All thats to say is this doesn't seem to be an issue with lots of elements in the table. I did however move the background color to be set in the `ClsutersTablePanel` component instead of here since the `CheckboxTable` is still being used in old VM